client/my-sites/customer-home/style.scss

```diff
@@ -238,6 +238,17 @@ body.is-section-home.theme-default.color-scheme {
 	margin: 32px 32px 0;
 }
 
+.primary__customer-home-location-content .dot-pager__control-choose-page,
+.primary__customer-home-location-content .dot-pager__control-prev,
+.primary__customer-home-location-content .dot-pager__control-next,
+.customer-home__card .dot-pager__control-choose-page,
+.customer-home__card .dot-pager__control-prev,
+.customer-home__card .dot-pager__control-next {
+	&:focus-visible {
+		box-shadow: 0 0 0 2px var( --color-primary );
```

Comment:
I think we can do this in `dot-pager` component itself. See https://github.com/Automattic/wp-calypso/blob/0ecb6b29ef6241aa938d62ec441e0e7794f5d59f/client/components/dot-pager/style.scss#L41

Reply:
That crossed my mind, but I'm not 100% sure that the customized outline is something unique to My Home or other pages as well, which is why I only added the override in the My Home page CSS. Does that make sense?

Reply:
I think our [_reset.scss](https://github.com/Automattic/wp-calypso/blob/0ecb6b29ef6241aa938d62ec441e0e7794f5d59f/client/assets/stylesheets/shared/_reset.scss#L128) removed the outline. So it's not customized outline and it's good to apply to component itself.